Watchman.Discord/Integration/DevscordFramework/CustomCommandsLoader.cs

```diff
@@ -35,13 +35,14 @@ public async Task<List<CustomCommand>> GetCustomCommands()
         public async Task InitDefaultCustomCommands()
         {
             var customCommands = new List<AddCustomCommandsCommand>();
-            foreach (var serverId in new ulong[] { 636238466899902504, 597066406521208852 })
+            foreach (var serverId in new ulong[] { 636238466899902504, 597066406521208852, 729754533311283293 })
```

Comment:
ten ostatni serwer na pewno ma zostać w kodzie?

Reply:
ewentualnie na nasz serwer można dodać więcej botów 
co myślisz @gagyn ?

Reply:
no można

Reply:
jeszcze usuń to ID z brancha i będzie dobrze

Reply:
zrobione